zebra-network/src/config.rs

```diff
@@ -33,8 +33,13 @@ pub struct Config {
     /// set size to reduce Zebra's bandwidth usage.
     pub peerset_initial_target_size: usize,
 
-    /// How frequently we attempt to connect to a new peer.
-    pub new_peer_interval: Duration,
+    /// How frequently we attempt to crawl the network to discover new peer
+    /// connections.
+    ///
+    /// This duration only pertains to the rate at which zebra crawls for new
+    /// peers, not the rate zebra connects to new peers, which is restricted to
+    /// CandidateSet::PEER_CONNECTION_INTERVAL
+    pub crawl_new_peer_interval: Duration,
```

Comment:
This is a breaking change, because we changed a config variable name.

I'll update the PR name so we notice in the release notes.

Reply:
I've fixed the PR to support backwards compatibility via `#[serde(alias = ..)]`

Reply:
Thanks, I updated the ticket title so we know it's a deprecation, not a breaking change.